dev-db/tarantool/metadata.xml

```diff
@@ -19,6 +19,7 @@
 	<flag name="backtrace">Support for detail backtraces</flag>
 	<flag name="gcov">Build with coverage support</flag>
 	<flag name="gprof">Build with profiling support</flag>
+	<flag name="embed-luarocks">Embed LuaRocks into tarantool binary</flag>
```

Comment:
Nit: Same here, it seems the flags are sorted alphabetically.

Reply:
Fixed